layer/logged_data/data_logging_request.py

```diff
@@ -0,0 +1,20 @@
+from typing import Any, Callable, Optional
+
+from layer.logged_data.retrying_files_sender import RetryingFilesSender
+
+
+class DataLoggingRequest:
+    def __init__(
+        self,
+        files_storage: RetryingFilesSender,
+        func: Callable[[], Optional[Any]],
```

Comment:
nit: let's have a better name here, smt like `queued_operation_func` or smt (same for other places where `func` is used)

Reply:
thx, renamed